docs_user/modules/openstack-mariadb_copy.adoc

```diff
@@ -55,6 +50,7 @@ endif::[]
 ifeval::["{build}" == "downstream"]
 MARIADB_IMAGE=registry.redhat.io/rhosp-dev-preview/openstack-mariadb-rhel9:18.0
 endif::[]
+STORAGE_CLASS=crc-csi-hostpath-provisioner
```

Comment:
This is an odd addition - is this appropriate?

Reply:
Yes, it does. The same pattern is shown at https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/tests/roles/ovn_adoption/tasks/main.yaml#L57 which is defined at https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/tests/roles/ovn_adoption/defaults/main.yaml#L2

Reply:
I think the value is CRC-specific and so they wouldn't apply to customers verbatim. We do say "The values are just illustrative, use values that are correct for your environment". Pedro used "local-storage" which is probably a better default for downstream.

Reply:
If you could post a separate PR with the docs fixes (on top of this PR content essentially), we could land this one without having to re-run CI i think.